samples/wifi/wfa_qt_app/src/qt_app_main.c

```diff
@@ -33,9 +33,12 @@ int init_usb(void)
 }
 #endif /* CONFIG_USB_DEVICE_STACK */
 
+int check_events(void);
```

Comment:
```suggestion
int wpa_supp_event_handler(void);
```

Reply:
The same name is being used, so modified to ``wpa_supp_events_init``

Reply:
add prefix `qt_`